backend/routes/post.py

```
@@ -247,120 +264,79 @@ async def create_document_post(
     db: Session = Depends(get_db)
 ):
     try:
-        # Combine Date & Time
-        local_datetime = datetime.strptime(f"{event_date} {event_time}", "%Y-%m-%d %H:%M")
-
-        # Convert to UTC
-        local_tz = ZoneInfo(user_timezone)
-        local_dt_with_tz = local_datetime.replace(tzinfo=local_tz)
-        event_datetime_utc = local_dt_with_tz.astimezone(ZoneInfo("UTC"))
-
+        dt_local = datetime.strptime(f"{event_date} {event_time}", "%Y-%m-%d %H:%M")
```

Ruff (0.8.2)

Naive datetime constructed using `datetime.datetime.strptime()` without %z

(DTZ007)